lte/gateway/c/session_manager/test/Matchers.h

```diff
@@ -179,8 +179,6 @@ MATCHER_P6(
       const auto& qos_info = cfg.rat_specific_context.lte_context().qos_info();
       if (!info.ambr) {
         return false;
-      } else if (info.ambr->max_bandwidth_ul() != qos_info.apn_ambr_ul()) {
-        return false;
```

Comment:
separate out session manager related changes to another PR.

Reply:
agree, above change unintentionally included part of this PR